controllers/photoController.js

```diff
@@ -2,8 +2,76 @@ const url = require("url");
 
 const Photo = require("../models/Photo");
 const Point = require("../models/Point");
+const Map = require("../models/Map");
+
 const { ERROR_MESSAGE } = require("../constants");
 
+const uploadPhoto = async (req, res, next) => {
+  const photo = JSON.parse(req.body.photo);
+  const { createdAt, createdBy } = photo;
+  const { description } = req.body;
+
+  const point = JSON.parse(req.body.point);
+  const { latitude, longitude, placeName } = point;
+
+  const { map } = req.body;
+
+  try {
+    const currentPoint = await Point.findOne({ latitude, longitude }).exec();
+    const currentMap = await Map.findById(map).exec();
```

Comment:
여기도 두 가지 DB 를 저장할 때처럼 순서대로 이행되지 않아도 되는 작업이니 병렬적으로 실행시켜주면 조금 더 효율적이게 될 수 있을 것 같습니다! ㅎㅎ

```suggestion
    const [currentPoint, currentMap] = await Promise.all([
      Point.findOne({ latitude, longitude }).exec(), 
      Map.findById(map).exec()
    ]);
```

Reply:
👍